src/main/java/org/jabref/gui/exporter/ExportToClipboardAction.java

```diff
@@ -55,9 +51,8 @@ public void run() {
             return;
         }
 
-        List<Exporter> exporters = Globals.exportFactory.getExporters();
-        exporters.sort(Comparator.comparing(Exporter::getDisplayName));
-        List<String> exportFormatDisplayNames = exporters.stream().map(Exporter::getDisplayName).collect(Collectors.toList());
+        List<Exporter> sortedExporters = Globals.exportFactory.getExporters().stream().sorted(Comparator.comparing(Exporter::getDisplayName)).collect(Collectors.toList());
```

Comment:
I think you can improve this a bit, if you return here only the stream, and reuse it for the displayName. Is more efficent then adding it to a list and iterating that list again.